bittensor/_dataset/dataset_impl.py

```diff
@@ -1,536 +1,490 @@
-""" Implementation for the dataset and GenesisTextDataset class, which handles dataloading from ipfs
+""" Implementation of Axon, services Forward and Backward requests from other neurons.
```

Comment:
Artefact from the other branch? I see axons mentioned in some of the docstrings as well